packages/dom/src/range-node-conversion.ts

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
Could just call this file `range.ts`.

Reply:
I thought about it, but it’s seems confusing with RangeSelector! (which now lives in a folder called `range`, so besides confusion it might create ambiguity).

Reply:
I may just call it `to-range.ts` for now, as that the name of its only export. (I started coding the inverse as well, `toNode`, hence I had opted for a more generic name)

I would be fine with throwing `ownerDocument` into the same file, if you like.